utils/handlers/graphql.go

```diff
@@ -50,3 +51,28 @@ func HandleGraphQLRequest(graphql *graphql.Module) http.HandlerFunc {
 	}
 
 }
+
+// HandleInspectionRequest creates the schema inspection endpoint
+func HandleInspectionRequest(schema *schema.Schema) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		// Create a context of execution
+		ctx, cancel := context.WithTimeout(r.Context(), 5*time.Second)
+		defer cancel()
+
+		vars := mux.Vars(r)
+		dbType := vars["dbType"]
+		col := vars["col"]
+		project := vars["project"]
+
+		result, err := schema.SchemaInspection(ctx, dbType, project, col)
```

Comment:
You should do some authentication before this step. Look at read config handler in `handlers/config.go`. we just want to check for a valid JWT token